### PR TITLE
[6840] Amended guidance text

### PR DESCRIPTION
**Pull request by defong**
### Context
guidance text

### Changes proposed in this pull request
Amended guidance text

### Guidance to review
https://educationgovuk.sharepoint.com.mcas.ms/:w:/r/sites/TeacherServices/_layouts/15/Doc.aspx?sourcedoc=%7BBFA890CD-19D9-44D3-80AF-0AEC807CA0DA%7D&file=About%20Register%20trainee%20teachers.docx&action=default&mobileredirect=true

https://register-pr-4190.test.teacherservices.cloud/guidance/about-register-trainee-teachers

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [ ] Do we need to send any updates to DQT as part of the work in this PR?
- [ ] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

**Comment by defong** [comment]
[Register guidance 'About' - content review and crit](https://trello.com/c/bDvjG6gB/6840-register-guidance-about-content-review-and-crit)

File: app/views/guidance/about_register_trainee_teachers.md [hunk 1]
```diff
@@ -8,18 +8,17 @@ Register trainee teachers (Register) is the service that Initial Teacher Trainin
 The DfE uses this trainee data to:
 
 * allocate the correct amount of funding to training providers and eligible trainees
-* create the ITT census publication
-* create the ITT performance profiles publication
+* <a href="/guidance/census-sign-off" class="govuk-link">create the ITT census publication</a>
+* <a href="/guidance/performance-profiles" class="govuk-link">create the ITT performance profiles publication</a>
```
[the remark this answers is not in the document]
Comment (defong) [reply app/views/guidance/about_register_trainee_teachers.md:12]: Needed the govuk styling

[the remark this answers is not in the document]
Comment (tomtrentham) [reply app/views/guidance/about_register_trainee_teachers.md:12]: Doesn't the `govuk-link` class automatically get added to links?

[the remark this answers is not in the document]
Comment (defong) [reply app/views/guidance/about_register_trainee_teachers.md]: Learn something new everyday

**Review by tomtrentham — APPROVED** [review]
:shipit:

**Pull request merged**